README.md

```diff
@@ -8,22 +8,18 @@ http://visualization.phillybuildingbenchmarking.com/#/
 
 ### Developing
 
-Dependencies:
-```
-# TODO: Add vagrant vm
-sudo apt-get install ruby1.9.1-dev
-sudo gem install sass compass
-sudo npm install -g grunt-cli bower
-npm install
-bower install
-```
-
-Then to serve the dev server:
-```
-grunt serve
-```
-
-The dev app will be served at localhost:9000 and automatically opened in a new tab in your default browser.
+#### Dependencies
+Docker
+
+#### Setup
+This project is containerized via Docker.
+
+Run `docker build .` from directory root
+
+#### Run
+Then run `docker run -p 9000:9000 <image_id>`
```

Comment:
```suggestion
Then run `docker run --rm -p 9000:9000 <image_id> grunt serve`
```
Sorry, apparently I left this out when I added the review!

Reply:
Yeah, this seems good. It served the site without it because of the `CMD` in the Dockerfile but I'll remove that to make it more flexible for serving/building, allowing the user to specify the command.